### ISSUE TITLE
Unable to run on Windows

### ISSUE THREAD

**Issue by Kaba2** [zinserjan/mocha-webpack#205]
After installing mocha-webpack on Windows, I am unable to run mocha-webpack from command-line: "'mocha-webpack' is not recognized as an internal or external command, operable program or batch file.".

**Comment by Kaba2** [comment]
The solution is to install mocha-webpack globally. Or, if mocha-webpack is installed locally, then to call it as node ./node_modules/mocha-webpack/bin/mocha-webpack.js.

**Comment by astorije** [comment]
@Kaba2, when you install `mocha-webpack` (or any package) locally, to run it, you need to either:

- Add it to a script [such as this one](https://github.com/thelounge/thelounge/blob/fc7e8339b759389ff6bccf13986e84a699298a88/package.json#L23) then call this script with `npm`/`yarn` CLI
- With Yarn directly, call `yarn mocha-webpack [args]`
- With npm directly, call `./node_modules/.bin/mocha-webpack [args]` (I believe `npx mocha-webpack [args]` would do the job too but I've never used it since I moved to Yarn)

In short, this is not related with this package :)
Would you mind closing this issue? 🙏
